site/source/docs/executing_scripts.md

```diff
@@ -0,0 +1,115 @@
+title: Executing Scripts
+layout: docs
+---
+
+There are various features in Embark that help you making the deployment of your DApps and Smart Contracts as smooth as possible. Next to general [Smart Contract Configurations](contracts_configuration.html), [Deployment Hooks](contracts_configuration.html#Deployment-hooks) and [Cockpit](cockpit_introduction.html), there's the possibility to run (migration) scripts as well.
+
+In this guide we'll explore why scripts are useful and how they can be run.
+
+## Why scripts?
+
+Given that Embark supports [afterDeploy](contracts_configuration.html#afterDeploy-hook) hooks that make it extremely easy to perform custom operations after all of your Smart Contracts have been deployed, you might wonder when and where scripts can be useful.
+
+It's important to note that `afterDeploy` hooks are executed every time all Smart Contracts have been deployed. Often there are cases where running a (migration) script manually is what you really need.
+
+Scripts let you exactly that as they can be run at any time, regardless of what your app's current deployment status is.
```

Comment:
You're missing a word after `let` I think

Reply:
Good catch!